### PR TITLE
jd 2021 09 more js tests

### PR DESCRIPTION
**Pull request by goapunk**

### CODE REVIEW

**Review by khamui — COMMENTED** [review]
Very nice! just some things that i found..

File: components/__tests__/ButtonCounter.test.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { render } from '@testing-library/react-native';
+import { ButtonCounter } from '../ButtonCounter';
+import Icon from 'react-native-vector-icons/SimpleLineIcons';
+
+test('Test ButtonCounter', async () => {
```
[the remark this answers is not in the document]
Comment (khamui) [reply components/__tests__/ButtonCounter.test.js:6]: and if you would make use of `findBy` if i understand correctly

[the remark this answers is not in the document]
Comment (goapunk) [reply components/__tests__/ButtonCounter.test.js:6]: the async was not needed, I just removed it

File: components/__tests__/ButtonSubmit.test.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react-native';
+import { ButtonSubmit } from '../ButtonSubmit';
+
+test('Test ButtonSubmit', async () => {
+  const onPress = jest.fn(() => {});
```
[the remark this answers is not in the document]
Comment (goapunk) [reply components/__tests__/ButtonSubmit.test.js:6]: It's not really needed, the onPress will just be undefined then, I guess that's ok. I removed the asyns

File: components/__tests__/TextSourceSans.test.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import { render } from '@testing-library/react-native';
+
+test('Test TextsourceSans no style in props', () => {
+  const { toJSON } = render(<TextSourceSans>{'TestText'}</TextSourceSans>);
+  expect(toJSON()).toMatchSnapshot();
+});
+
+test('Test TextsourceSans no style in props', () => {
+  const { toJSON } = render(
+    <TextSourceSans style={{ fontSize: 10 }}>{'TestText'}</TextSourceSans>
```
[the remark this answers is not in the document]
Comment (goapunk) [reply components/__tests__/TextSourceSans.test.js:13]: done

**Comment by goapunk** [comment]
@kha @phillimorland I addressed the comments and pushed the changes

**Comment by goapunk** [comment]
tests are failing with #187 now, will have to investigate the reason

**Comment by goapunk** [comment]
ok, should work again, but should be merged after #187

**Review by phillimorland — APPROVED** [review]
very nice! running locally, feel free to merge once conflicts resolved

**Pull request merged**